recipes/pyobjc-framework-coreservices/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "pyobjc-framework-CoreServices" %}
+{% set version = "8.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 58bad844c8e5ee82852d860b117d1238c3ed2db8dd57f754feb6bd5801fe878d
+  patches:
+    - patches/0001-Remove-werror-from-compile-flags.patch
+
+build:
+  skip: true   # [not osx]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pyobjc-core {{ version }}
+    - pyobjc-framework-fsevents {{ version }}
```

```suggestion
    - pyobjc-core >={{ version }}
    - pyobjc-framework-fsevents >={{ version }}
```
The upstream version spec is >= not ==.

https://github.com/ronaldoussoren/pyobjc/blob/26f2a5dfb8a52baf851b2890267e27a916b1f3f8/pyobjc-framework-CoreServices/setup.py#L35